src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs

```diff
@@ -108,6 +119,8 @@ private void StopServiceCallback(object sender, EventArgs e)
         {
             if (_stopMenuCmd.Enabled)
```

Comment:
Can this flag be reset during test execution? If so then I would use a separate flag to track if we added `CurrentDomainAssemblyResolve`

Reply:
I don't think so, but I also don't think this command is ever used. We only enable the assembly resolve handler when starting integration tests, so it's outside any user path.